hr_timesheet_day_week/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# © 2022 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+{
+    "name": "Add Weekdays to time sheets",
+    "description": "Add Weekdays to time sheets",
+    "category": "Timesheet",
+    "version": "15.0.1.0.0",
+    "depends": ["hr_timesheet"],
+    "data": [
+        "views/hr_timesheet_view.xml",
+    ],
+    "author": "Solvos Consultoría Informática",
+    "website": "https://github.com/solvosci/slv-timesheet",
+    "license": "AGPL-3",
```

Comment:
Use LGPL by default, if no OCA module involved:
```suggestion
    "license": "LGPL-3",
```